src/posts/en/configuring-babel-6-for-node-js.md

```diff
@@ -0,0 +1,164 @@
+
+#### Using Source maps in your file
+If you are thinking that's all cool and fun but what about syntax highlighting and real time code debugging. You don't have to be worried. Source maps are just for this purpose. Source maps tell node.js that this code is transpiled and let's you highlight **source** file instead of the **transpiled** file 😄. YAYY!
+
+This file here `code/error.js` throws an error after the second yield in the generator but the transpiled code doesn't quite look like this.
+```js
+function* errorFulGenerator() {
+  yield "yo"
+  throw new Error("source maps are awesome")
+  return ""
+}
+
+var errorGen = errorFulGenerator()
+errorGen.next()
+errorGen.next()
+```
+
+We use this command to generate **source maps** along with the **transpiled** code *notice the `--source-maps flag`*
+```bash
+babel code/ -d build/ --source-maps
+```
+
+Now when we encounter the error we get useful debugging such as this
+```js
+errorGen.next()
+         ^
+Error: source maps are awesome
+```
+So this is how you'll use source maps
+
+#### Setting up npm command
+In order to simplify the build process even more. You can update your `package.json` file to include a build script for babel. In `package.json` script object you can add a build script such as the one below
+```json
+"scripts": {
+  "build": "babel -w code/ -d build -s"
+}
+```
+Now, we can run
+```bash
+npm run build
+```
+and get all the ES6/ES7 goodness instantly today. :)
+
+
+#### Learn more about Babel
+This is a basic tutorial on babel but the Babel world just starts here. It's surrounded by a wonderful community and is used by big names in IT world. Babel has support for all the major build tools too such as [GruntJS](https://www.npmjs.com/package/grunt-babel) and [GulpJS](https://npmjs.org/package/gulp-babel/). You can check them all out here on [Babel Website](https://babeljs.io/docs/setup/)
+
+These are some of the resources that can even up your game further in the babel world
+- [Learn ES6 and Babel using this detailed tutorial](http://ccoenraets.github.io/es6-tutorial/index.html)
+- [Read the Babel docs on setting up Babel (They're helpful)](https://babeljs.io/docs/setup/)
+
+
+##### Source code and Contributions and Thank yous
+Source code for this tutorial is available in this [repo](https://github.com/abdulhannanali/babel-configuration-tutorial).
+
+If you find some typo or would like to make some update. Please do so using the power of issues and PR in our [Github Repo](https://github.com/abdulhannanali/babel-configuration-tutorial).
+
+I would also like to Thank [Fabrício Matté](http://ultcombo.js.org/) for approving this article to be posted on JSRocks and the corrections he made.
```

Comment:
Thank -> thank
JSRocks -> JS Rocks (that is how it appears in the site title)
Also, I don't think the statement is entirely necessary, but you can leave it if you like. :smile: